src/main/features/claimant-response/views/defendants-response.njk

```diff
@@ -57,16 +64,52 @@
 {% block content %}
   <div class="grid-row">
     <div class="column-two-thirds">
-      {% if claim.response.responseType === ResponseType.FULL_ADMISSION.value %}
-        <p>{{ t('{{ defendantName }} admits they owe all the money you’ve claimed.', { defendantName: defendantName }) }}</p>
-        <p>
-          {{
-            paymentIntentionFragment(
+
+      {% if alreadyPaid %}
+
+
+        {% if partiallyPaid %}
+          <p>{{ t('{{ defendantName }} said they paid you {{ amount }}. They said this is all they owe, not the amount you claimed.', { defendantName: defendantName, amount: claim.response.amount | numeral }) }}</p>
+
+          <h3 class="heading-small">{{ t('When they paid this amount') }}</h3>
+          <p>{{ claim.response.paymentDeclaration.paidDate | date }}</p>
+
+          <h3 class="heading-small">{{ t('How they say they paid') }}</h3>
+          <p>{{ claim.response.paymentDeclaration.explanation }}</p>
+
+          <h3
+            class="heading-small form-group-related">{{ t('Why they say they don’t owe the amount you claimed') }}</h3>
+          <p>{{ t('"{{ defence }}"', { defence: claim.response.defence }) }}</p>
+
+          {% if claim.response.timeline and claim.response.timeline.rows.length > 0 %}
+            <h3 class="heading-small form-group-related">{{ t('Their timeline of events') }}</h3>
+            {{ timelineFragment(claim.response.timeline) }}
+          {% endif %}
+
+          {% if claim.response.evidence and claim.response.evidence.comment %}
+            <h3 class="heading-small form-group-related">{{ t('They disagree with your evidence') }}</h3>
+            <p>{{ t('"{{ comment }}"', { comment: claim.response.evidence.comment }) }}</p>
+          {% endif %}
+        {% else %}
```

Comment:
This else block is duplicate from line 72 to 78. please remove else block and move this common code outside if block